capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -0,0 +1,21 @@
+document.body.prepend(dynamicButton);
+
+function createClassroom() {
+    var auth2 = gapi.auth2.getAuthInstance();
+    var name = "";
+    var email = "";
+    if (auth2.isSignedIn.get()) {
+        name = auth2.currentUser.get().getBasicProfile().getName();
+        email = auth2.currentUser.get().getBasicProfile().getEmail();
+    }
+
+    var subject = document.querySelector('#subject').value;
```

Comment:
Having a reference to a particular DOM structure right in the method reduces the reusability of this function to just this page. Instead you could pass the subject as a parameter to the function, and this function suddenly becomes re-usable by potentially any part of the app.

Reply:
Hmm sounds good!

Reply:
got it